qmetaobject/src/qtdeclarative.rs

```diff
@@ -502,16 +502,105 @@ pub fn qml_register_type<T: QObject + Default + Sized>(
     })
 }
 
-/// Wrapper around [`void qmlRegisterModule(const char *uri, int versionMajor, int versionMinor)`][qt] function.
```

Comment:
Something's funny about this diff.

Reply:
Yeah, it's thinking I'm changing the qml_register_module. I'm not sure if there's a way to convince git not to think that.